modules/sharedaddy/sharing-sources.php

```diff
@@ -391,9 +391,7 @@ function sharing_twitter_via( $post ) {
 
 		// Strip out anything other than a letter, number, or underscore.
 		// This will prevent the inadvertent inclusion of an extra @, as well as normalizing the handle.
-		$twitter_site_tag_value = preg_replace( '/[^\da-z_]+/i', '', $twitter_site_tag_value );
-
-		return $twitter_site_tag_value;
+		return preg_replace( '/[^\da-z_]+/i', '', $twitter_site_tag_value );
```

Comment:
Lines are a bit out of order between here and WPCOM as to when the `jetpack_sharing_twitter_via` filter is applied. I think I prefer what you have here (with the comment above the actual stripping).